Makefile

```diff
@@ -115,6 +114,9 @@ GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint
 golangci-lint: ## Download golangci-lint locally if necessary.
 	$(call go-get-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1)
 
+lint: golangci-lint ## Run go lint against code.
+	$(GOLANGCI_LINT) run -v
```

Comment:
It seems that we will install golangci-lint@1.42.1.
But we run golangci-lint@1.31.1 in github actions.
We should make them the same version.

Reply:
fixed.